docs/basics/acct-team-mgmt/switching-active-team.md

```diff
@@ -8,6 +8,12 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <p><span className="sauceGreen">Enterprise Plans only</span> <span className="sauceYellow">Beta feature</span></p>
 
+:::caution
+This functionality *affects* tests run via *API* calls. It will result in tests being assigned to current active team.
+
+Please be cautious when using it, currently we *recommend* using it mostly for UI related work.
```

Comment:
"Please be cautious when using it. Currently, we *recommend* using it mostly for UI-related work."